### DISCUSSION TITLE
Regression in @types/node

### DISCUSSION THREAD

**Discussion by dbrockman** [DefinitelyTyped/DefinitelyTyped#54738] [Issues with a @types package]
The method [`write`](https://nodejs.org/api/fs.html#fs_filehandle_write_buffer_offset_length_position) disappeared from the type `FileHandle` from `fs/promises` after updating `@types/node` to `16.4.3`.

**Comment by KapitanOczywisty** [comment]
It is indeed missing. As I understand definitions were auto-generated #54417 @SimonSchick

**Reply by SimonSchick** [reply]
That's only for the docs, but during the process of generating the docs I discovered many incorrect/non-existant declarations, I think this one got accidentally deleted (in the crossfire so to speak).
I will bring it back shortly.

**Comment by SimonSchick** [comment]
Marking this as answered, cl should (hopefully) land today.
